assemblies/pentaho-solutions/src/main/resources/pentaho-solutions/system/applicationContext-spring-security-ssrf.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--+
+  | Application context containing SSRF related beans.
+  +-->
+<beans xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns="http://www.springframework.org/schema/beans"
+       xsi:schemaLocation="
+          http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-4.3.xsd"
+       default-lazy-init="true">
+
+    <!-- read the alternate-fully-qualified-server-url and fully-qualified-server-url from the system.properties file -->
```

Comment:
Should be `alternative-fully-qualified-server-url`.

Reply:
addressed